core-library/src/main/java/org/silverpeas/core/admin/domain/DomainDriverManager.java

```diff
@@ -234,8 +229,7 @@ public void updateUserFull(UserFull user) throws Exception {
       // index informations relative to given user
       indexUser(user.getId());
     } catch (AdminException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1166)